common/txmgr/types/config.go

```diff
@@ -10,30 +10,37 @@ type TxmConfig[FEE_UNIT Unit] interface {
 	ReaperConfig
 
 	SequenceAutoSync() bool
-	UseForwarders() bool
-	MaxQueuedTransactions() uint64
+}
+
+type TxmTransactionsConfig interface {
```

Comment:
Maybe s/Transaction/Tx all around? :thinking: 
```suggestion
type TxmTxsConfig interface {
```

Reply:
I would rather keep it as Transactions as all of the other interfaces use the same pattern (using the antecedent accessor in the config var name).

Also my brain finds it hard to parse TxmTxs :P

Reply:
But then why not this?
```suggestion
type TransactionManagerTransactionsConfig interface {
```

Reply:
Ha, good point! I've addressed